includes/restore.js

```diff
@@ -27,6 +29,11 @@ module.exports = function(dbUrl, buffersize, parallelism, readstream, callback)
     var liner = require('../includes/liner.js');
     var writer = require('../includes/writer.js')(dbUrl, buffersize, parallelism);
 
+    writer.on('error', function(err) {
```

Comment:
Just eyeballing this again on the train, we already have a writer 'error' listener in app.js. I reckon we can remove this and make the existing one call `callback(e, null);`. What say?